client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -470,6 +470,26 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: <JetpackLogo className="gridicon" size={ 18 } />,
 		},
+		{
+			name: 'manageJetpackModules',
+			label: __( 'Manage Jetpack modules' ),
+			callback: setStateCallback(
+				'manageJetpackModules',
+				__( 'Select site to manage Jetpack modules' )
+			),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `${ site.URL }/wp-admin/admin.php?page=jetpack_modules` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic || isNotAtomicJetpack( site ),
+				filterNotice: __(
+					'Only listing sites with hosting features enabled or self-hosted sites.'
```

Comment:
I'd suggest `Only listing sites with Jetpack modules available`.

This PR also needs to apply the changes from https://github.com/Automattic/wp-calypso/pull/85600.

Reply:
That's an option!

One thing that I am worried about is that it might be confusing for the users since Jetpack menu tab is present also on sites on a Free plan:

<img width="472" alt="Screenshot 2023-12-21 at 9 05 40 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/7d1b7b16-c1f2-452b-8116-7d0110ac5be4">

Reply:
Following the same logic, let's use `Only listing sites with Jetpack modules available` and `No sites with Jetpack modules available.`

I was thinking about another message referring about the possibility to install plugins , something like `Listing only sites where plugins are available.`. We can revisit in next iteration.
On argument in favor of reusing the same message is that we can add the messages and filter on `siteFilters`.